docs/reference/index-modules/blocks.asciidoc

```diff
@@ -37,7 +37,8 @@ block and makes resources available almost immediately.
 +
 IMPORTANT: {es} adds and removes the read-only index block automatically when
 the disk utilization falls below the high watermark, controlled by
-<<cluster-routing-flood-stage,cluster.routing.allocation.disk.watermark.flood_stage>>.
+<<cluster-routing-flood-stage,cluster.routing.allocation.disk.watermark.flood_stage>>
+and <<cluster-routing-flood-stage,cluster.routing.allocation.disk.watermark.flood_stage.max_headroom>>.
```

Comment:
Hm, shouldn't this refer to the high watermark instead of the flood watermark and the respective headroom?

Reply:
Oh good catch, there was already some misalignment in master here. But I believe the text wanted to refer to the flood watermark, as the read-only block I believe is added when a node passes the flood watermark. Please correct me if I am wrong.

Reply:
You are right, that was a pre-existing issue, on a second thought both the text and the reference are misalligned. To the best of my knowledge:
- `es` **adds** the read-only index block when the disk usage goes over the flood stage watermark, but
- it **removes** the read-only block when the disk usage falls under the high watermark. 
So both watermarks play a role in this sentence :p.

Could you rephrase this to be more clear?

Reply:
Oh, I was not aware it gets removed when it falls under the high watermark, thanks for explaining! Just pushed a commit that clarifies it. Please tell me if it needs more clarifications.